### ISSUE TITLE
Single node swarm cluster instead of compose

### ISSUE THREAD

**Issue by aminvakil** [getsentry/self-hosted#1674]
### Problem Statement

`docker-compose` is getting worse with each release and can't be relied on even for testing products as it will break more and more.

### Solution Brainstorm

Maybe we can change the current situation to use a single node swarm cluster.

**Comment by madson7** [comment]
@aminvakil This issue has an update?

**Comment by aminvakil** [comment]
@madson7 Nope, but AFAICS docker-compose has fixed lots of its problems since then.

**Comment by madson7** [comment]
I will continue with a self-hosted fork to accompany the main project, I created a [feature branch](https://github.com/madson7/self-hosted/tree/feature/swarm) to customize the installation and update via swarm.

Sorry, I'm new and I don't know the best way to contribute to the project here, if you have any tips or criticisms, feel free.

**Comment by aminvakil** [comment]
> I will continue with a self-hosted fork to accompany the main project, I created a [feature branch](https://github.com/madson7/self-hosted/tree/feature/swarm) to customize the installation and update via swarm.
> 
> Sorry, I'm new and I don't know the best way to contribute to the project here, if you have any tips or criticisms, feel free.

Sorry for late response, I'm not sure if there is interest in changing current state with docker-compose by maintainers.

**Comment by aldy505** [comment]
I'm closing this issue as we'll stay for Docker Compose until things break :)
